### PR TITLE
Support Vuejs file (fix #6290)

### LINKED ISSUE #6290: enable emmet in vuejs file ".vue"
- VSCode Version:
- OS Version:

Steps to Reproduce:

1. 
2.

### PR DESCRIPTION
**Pull request by Ikuyadeu**
fix #6290.
[Vuejs file](http://vuejs.org/guide/application.html#Single-File-Components) is like HTML.
And this related #6191 and #4700.

### CODE REVIEW

**Comment by be5invis** [comment]
@yyx990803 Is that really appropriate?

**Comment by yyx990803** [comment]
I guess it doesn't hurt as long as it doesn't interfere when the user explicitly installs https://github.com/LiuJi-Jim/vscode-vue

File: src/vs/languages/html/common/html.contribution.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import ConfigurationRegistry = require('vs/platform/configuration/common/configu
 
 ModesRegistry.registerCompatMode({
 	id: 'html',
-	extensions: ['.html', '.htm', '.shtml', '.mdoc', '.jsp', '.asp', '.aspx', '.jshtm'],
```
Comment (Tyriar) [inline src/vs/languages/html/common/html.contribution.ts]: Is it a problem that .xhtml isn't here?

Comment (Ikuyadeu) [inline src/vs/languages/html/common/html.contribution.ts]: @Tyriar Thank you. Now, I added `.xhtml`.

**Comment by aeschli** [comment]
I tested the fix togther with the https://github.com/LiuJi-Jim/vscode-vue extension.
Unfortunately it seems HTML wins. Therefor I reverted the fix again.
Users can always associate the vue extension the html in the file association setting.

**Pull request merged**